docker-compose.yml

```diff
@@ -1,32 +1,62 @@
 # Note: Requires docker 1.10.0+ and docker-compose 1.6.0+.
-version: "2"
+version: '3.4'
 services:
   db:
     image: postgres:9.4
 
   # run the site with the django dev-server
```

Comment:
I think this comment doesn't make sense anymore.

Reply:
good point :)